src/schema/v2/homeView/__tests__/HomeViewSection.test.ts

```diff
@@ -37,9 +37,7 @@ describe("HomeViewSection", () => {
       ]
 
       const context = {
-        authenticatedLoaders: {
-          meLoader: jest.fn().mockReturnValue({ type: "User" }),
-        },
+        accessToken: "424242",
```

Comment:
We weren't actually using the me loader; so this has been replaced with a simple check for an access token.